### PR TITLE
Updates to game-service logic

### PR DESCRIPTION
**Pull request by LiamMoMurray**
Added portion of the gameService logic

### CODE REVIEW

File: game-service/src/main/java/com/lms/gameservice/controller/GameController.java
Comment (olanhealy) [inline game-service/src/main/java/com/lms/gameservice/controller/GameController.java:68]: keep spelling as 'authorisation' what has been done before

File: game-service/src/main/java/com/lms/gameservice/controller/GameController.java
Comment (olanhealy) [inline game-service/src/main/java/com/lms/gameservice/controller/GameController.java:86]: Not need to verify logged in? who calls process round? will system call will it be a cron job not really sure

**Review by olanhealy — CHANGES_REQUESTED** [review]
Has this also been tested before pr made? A description on what happens, etc would be needed before approval

**Comment by olanhealy** [comment]
Is this and marks coming in together now? Failing the u
Job atm

**Review by CaoimhinHawk1 — APPROVED** [review]
Looks good to me, approving since its passing unit tests now, good work Liam, get back to Olan on his review and get this pushed ASAP!

**Pull request merged**
